### ISSUE TITLE
1.5.0-beta13. Android: Cant change network connection

### ISSUE THREAD

**Issue by TikhomirovSergey** [appium/appium#6113]
Server: 1.5.0-beta13

Client code samples:

```java
    AppiumServiceBuilder builder = AppiumDriverLocalService.buildDefaultService();
    service.start();

    File appDir = new File("src/test/java/io/appium/java_client");
    File app = new File(appDir, "ApiDemos-debug.apk");
    DesiredCapabilities capabilities = new DesiredCapabilities();
    capabilities.setCapability(MobileCapabilityType.BROWSER_NAME, "");
    capabilities.setCapability(MobileCapabilityType.DEVICE_NAME, "Android Emulator");
    capabilities.setCapability(MobileCapabilityType.APP, app.getAbsolutePath());
    capabilities.setCapability(MobileCapabilityType.NEW_COMMAND_TIMEOUT, 120);
    driver = new AndroidDriver<WebElement>(service.getUrl(), capabilities);

    ...
    //airplane mode
    NetworkConnectionSetting networkConnection = new NetworkConnectionSetting(true, false, false);
    driver.setNetworkConnection(networkConnection);
```

Error which is thrown on the client side:
https://gist.github.com/TikhomirovSergey/6267cef8b3befba1f5fe
The session completely died.

Server output:
https://gist.github.com/TikhomirovSergey/28c2c494f8710f008988

Server output of node 1.4.x:
https://gist.github.com/TikhomirovSergey/955a6e40369dd40046a3

**Comment by TikhomirovSergey** [comment]
Same thing happens when 
```java
driver.toggleLocationServices();
```

**Comment by TikhomirovSergey** [comment]
appium/java-client#302
appium/appium-dotnet-driver#136

**Comment by scottdixon** [comment]
Resolved & published as `appium-android-driver@1.6.10`. Please confirm @TikhomirovSergey
